Mathlib/Data/ZMod/Basic.lean

```diff
@@ -456,6 +456,32 @@ def ringEquivCongr {m n : ℕ} (h : m = n) : ZMod m ≃+* ZMod n := by
           rw [Fin.coe_cast, Fin.val_add, Fin.val_add, Fin.coe_cast, Fin.coe_cast, ← h] }
 #align zmod.ring_equiv_congr ZMod.ringEquivCongr
 
+@[simp] lemma ringEquivCongr_refl (a : ℕ) : ringEquivCongr (rfl : a = a) = .refl _ := by
+  cases a <;> rfl
+
+lemma ringEquivCongr_refl_apply {a : ℕ} (x : ZMod a) : ringEquivCongr rfl x = x := by
+  rw [ringEquivCongr_refl]
+  rfl
+
+lemma ringEquivCongr_symm {a b : ℕ} (hab : a = b) :
+    (ringEquivCongr hab).symm = ringEquivCongr hab.symm := by
+  subst hab
+  cases a <;> rfl
+
+lemma ringEquivCongr_symm_apply {a b : ℕ} (hab : a = b) (y : ZMod b) :
+    (ringEquivCongr hab).symm y = ringEquivCongr hab.symm y := by
+  rw [ringEquivCongr_symm]
```

Comment:
This is simply `DFunLike.congr_fun (ringEquivCongr_symm hab) y` so I think we don't need it ...

Reply:
I wrote it because I thought I needed it in the application but indeed I didn't.